cfpb/capital-framework
# Adding lockfile to the project

[Latest NPM create a lockfile](http://blog.npmjs.org/post/161081169345/v500) during install. Adding it here.

## Additions

- NPM Lockfile

## Testing

- Run `npm install`, check that the lockfile didn't change (@contolini, how will we handle changes if that happens between PR submission and testing?)
- Run `npm run cf-link` (This has a side effect of taking care of the sub-dependencies in each component `package.json`)
- Run `gulp build` Ensure it builds without errors

## Review

- @contolini 

## Notes

- cfgov-refresh now requires the newest versions of Node and NPM. Upgrading other projects so there is parity.

## Checklist
